example_dags/example_snowflake_partial_table.py

```diff
@@ -8,39 +8,73 @@
 from astro.sql import append, load_file, transform
 from astro.sql.table import Table
 
```

Comment:
Perhaps this filename could include "append", so it is clear for users looking at example filenames a feature that is tested here.